assets/js/admin/form-builder.js

```diff
@@ -1451,6 +1451,22 @@ jQuery( function ( $ ) {
 		}
 	});
 
+	$( document ).on( 'change', '.evf-content-email-settings .evf-toggle-switch input', function(e) {
+		var $this = $( this ),
+			value = $this.prop('checked');
+
+			if ( value === false ) {
+				$this.closest('.evf-content-email-settings').find('.email-disable-message').remove();
+				$this.closest('.evf-content-section-title').siblings('.evf-content-email-settings-inner').addClass('everest-forms-hidden');
+				$('<p class="email-disable-message everest-forms-notice everest-forms-notice-info">Enable Email to send email notifications.</p>').insertAfter( $this.closest('.evf-content-section-title' ));
+			} else if( value === true ){
```

Comment:
Yoda follows in JS code too. Soon we will be straight forward for coding standard in JS files.

Reply:
Done :)